requirements.txt

```diff
@@ -1,2 +1,4 @@
+pillow
```

Comment:
Where did we used pillow?

Reply:
Here:
https://github.com/imjoy-team/jupyter-elfinder/blob/f0a4ad0e56b3c053b4bf80fbd1a16ccbc9b47229/jupyter_elfinder/elfinder.py#L1421